tests/e2e/e2e_test.go

```diff
@@ -104,7 +108,16 @@ func (ts *E2ETestSuite) startServer(extraArgs ...string) {
 	serverSigs := make(chan os.Signal)
 	serverDone := make(chan struct{})
 	go func() {
-		server.ServerMain("contest", args, serverSigs)
+		pc := server.PluginConfig{
+			TargetManagerLoaders: []target.TargetManagerLoader{
+				targetlist.Load,
+				targetlist_with_state.Load,
+			},
+			TestFetcherLoaders: []test.TestFetcherLoader{literal.Load},
+			TestStepLoaders:    []test.TestStepLoader{cmd.Load, sleep.Load},
+			ReporterLoaders:    []job.ReporterLoader{targetsuccess.Load, noop.Load},
+		}
+		_ = server.Main(&pc, "contest", args, serverSigs)
```

Comment:
You should not throw away the error here, this should have said "error duplicate user function" :)

Reply:
right. And it was already ignored in the original code, I just added `_ = ` to make the linter happy

Reply:
done, now the e2e tests are surfacing errors in the test report instead of ignoring them